src/components/OpenSource/Talks.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+import remcalc from 'remcalc'
+
+import BlueBackground from '../Common/BlueBackground'
+import CompactVideoLink from '../Common/CompactVideoLink'
+import StyledLink from '../Common/StyledLink'
+import { Grid, Row, Col } from '../grid'
+import { SectionTitle } from '../Typography'
+
+const StyledImage = styled.img`
+  padding: ${remcalc(83)} 0 ${remcalc(15)};
+`
+
+const StyledSectionTitle = styled(Col)`
+  padding: 0 0 ${remcalc(35)};
+
+  ${breakpoint('smallTablet')`
+    padding-left: ${remcalc(21)};
+  `}
+
+  ${breakpoint('tablet')`
+    padding-left: ${remcalc(24)};
+  `}
+`
+
+const StyledTalksRow = styled(Row)`
+  padding-bottom: ${remcalc(35)};
+`
+
+const StyledTalksLink = styled(StyledLink)`
+  padding-bottom: ${remcalc(90)};
+`
+
+const TalkCol = ({
+  title,
+  type,
+  link,
+  cta,
+  featured,
+  additionalInfo,
+  colorCode
+}) => (
+  <CompactVideoLink
+    key={title}
+    href={link}
+    themeVariation="dark"
+    mode="standalone"
+  >
+    {title}
+  </CompactVideoLink>
+)
+
+const TalksSection = ({ icon, title, talks, ctaText, ctaLink }) => (
+  <BlueBackground>
+    <Grid>
+      <StyledImage src={icon.file.url} alt={icon.title} />
+      <Row>
+        <StyledSectionTitle width={[1, 1, 1, 1, 1 / 2, 1 / 2, 1 / 2]}>
+          <SectionTitle reverse>{title}</SectionTitle>
+        </StyledSectionTitle>
+      </Row>
+      <StyledTalksRow>
+        {talks && talks.length && talks.map(talk => TalkCol(talk))}
```

Comment:
Can you put this `talks &&....` defensive coding around the whole `TalksSection` on the `open-source.js` file? If there are no talks at all coming from Contentful, we don't want to render any of this section at all. This code would leave the section title & CTA and simply render no videos, which is not quite what we want

Reply:
YES!!! Awesome idea, the parent should decide. Also I did it for partners as I used a similar logic. Great stuff Jen! Done.